src/components/JUPTable/JUPTable.tsx

```diff
@@ -21,18 +21,20 @@ import { DefaultTableRowsPerPage, DefaultTransitionTime, TableRowsPerPageOptions
 import { IHeadCellProps, ITableRow } from ".";
 
 interface ITableTitleProps {
-  title: string;
+  title?: string;
   path?: string;
   DisplayedComponents?: Array<React.ReactElement>;
 }
 
 const TableTitle: React.FC<ITableTitleProps> = ({ title, path, DisplayedComponents }) => {
   return (
     <>
-      <TitleText variant="h6" id="tableTitle">
-        {path && <SLink href={path}>{title}</SLink>}
-        {DisplayedComponents}
-      </TitleText>
+      {title && (
+        <TitleText variant="h6" id="tableTitle">
+          {path && <SLink href={path}>{title}</SLink>}
+          {DisplayedComponents}
```

Comment:
these changes are good if there is no case where DisplayedComponents is provided but no title is provided. I'm unsure if such a case exists, FYI.

Reply:
for now there's not a situation like that